src/components/table/__spec__.js

```diff
@@ -931,6 +930,21 @@ describe('Table', () => {
       expect(parent.className).toEqual('carbon-table foo');
     });
 
+    it('renders a caption tag when a caption prop is given', () => {
+      let wrapper = render(
```

Comment:
So you need to use `render` for this case? Would this work with `shallow` - curious